recipes/blue/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "blue" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/blue-{{ version }}.tar.gz
+  sha256: da06d09f8417ad475e21a26dd0a7dae6771146fdd7f5760450183183230f9ef1
+
+build:
+  entry_points:
+    - blue=blue:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - tox
+  run:
+    - black ==22.1.0
+    - flake8 >=3.8
+    - python >=3.6
+
+test:
+  imports:
+    - blue
+  commands:
+    - pip check
+    - blue --help
+  requires:
+    - pip
+
+about:
```

Comment:
Please fix this section. The summary is not helpful and the dev_url key has copypasta.

Reply:
Whops, trusted grayskull too much there, thanks for noticing.